src/SDKs/CognitiveServices/dataPlane/Vision/CustomVision/Prediction/Microsoft.Azure.CognitiveServices.Vision.CustomVision.Prediction.csproj

```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Vision.CustomVision.Prediction</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services CustomVision Prediction APIs.</Description>
-    <Version>0.11.0-preview</Version>
+    <Version>0.12.0-preview</Version>
```

Comment:
@areddish missing release notes.

Reply:
Done, the existing release notes still hold as there was a change that was made, reversed, and made again that was never released. I updated with the only other change to this client (naming).